StackExercise/MyStack.cs

```diff
@@ -0,0 +1,68 @@
+﻿using System.Drawing;
+
+namespace StackExercise;
+
+public class MyStack<T>
+{
+    private List<T> _myStack;
+
+    public MyStack()
+    {
+        _myStack = new List<T>();
+    }
+    
+    public void Push(T item)
+    {
+        _myStack.Add(item);
+    }
+    
+    public T? Pop()
+    {
+        if (_myStack.Count > 0)
+        {
+            var lastItemIndex = _myStack.Count - 1;
```

Comment:
what method on List<T> might be useful to get the last element? and what advantage or disadvantage using it would add?

Reply:
> what method on List might be useful to get the last element? and what advantage or disadvantage using it would add?

The better approach would be to use Last/LastOrDefault from the LINQ library, and since the List implements the IEnumerable it can be used directly on the List. The advantage is it would be much faster than using ElementAt, which iterates through the list. (Note: Last/LastOrDefault iterates through elements in other types of collections but not of type IList).
I've done this exercise before diving into the LINQ topic, that's why I used the wrong LINQ operation here.